app/src/main/java/ru/faimizufarov/simbirtraining/java/presentation/ui/screens/news/composeunit/NewsItem.kt

```diff
@@ -0,0 +1,180 @@
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat
+import com.bumptech.glide.integration.compose.ExperimentalGlideComposeApi
+import com.bumptech.glide.integration.compose.GlideImage
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toLocalDateTime
+import kotlinx.datetime.todayIn
+import ru.faimizufarov.simbirtraining.R
+import ru.faimizufarov.simbirtraining.java.domain.models.News
+
+@OptIn(ExperimentalGlideComposeApi::class)
+@Suppress("ktlint:standard:function-naming")
+@Composable
+fun NewsItem(
+    modifier: Modifier = Modifier,
+    news: News,
+    clickItem: (News) -> Unit,
+) {
+    val imagePath =
+        if (news.newsImages.first().contains("http")) {
+            news.newsImages.first()
+        } else {
+            "file:///android_asset/${news.newsImages.first()}"
+        }
+
+    Column(
+        modifier =
+            modifier
+                .background(colorResource(id = R.color.white))
+                .clickable { clickItem.invoke(news) },
+    ) {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            GlideImage(
+                model = imagePath,
+                contentDescription = null,
+                modifier = modifier,
+                contentScale = ContentScale.Fit,
+            )
+            Image(
+                painter = painterResource(id = R.drawable.fade),
+                contentDescription = null,
+                modifier = Modifier.fillMaxWidth(),
+                contentScale = ContentScale.Crop,
+            )
+            Text(
+                modifier =
+                    modifier
+                        .fillMaxWidth()
+                        .align(Alignment.BottomCenter)
+                        .padding(start = 40.dp, end = 40.dp)
+                        .offset(y = 44.dp),
+                text = news.nameText,
+                color = colorResource(id = R.color.blue_grey),
+                fontSize = 21.sp,
+                fontFamily = FontFamily(Font(R.font.officina_sans_extra_bold_c)),
+                textAlign = TextAlign.Center,
+            )
+        }
+        Image(
+            modifier =
+                modifier
+                    .padding(top = 52.dp)
+                    .align(Alignment.CenterHorizontally),
+            painter = painterResource(id = R.drawable.decor),
+            contentDescription = null,
+        )
+        Text(
+            modifier =
+                modifier
+                    .fillMaxWidth()
+                    .padding(
+                        top = 10.dp,
+                        start = 24.dp,
+                        end = 24.dp,
+                        bottom = 16.dp,
+                    ),
```

Comment:
Просто сахар для читаемости
```suggestion
                    .padding(top = 10.dp, bottom = 16.dp)
                    .padding(horizontal = 24.dp),
```

Reply:
к такому коду Я и стремлюсь)